src/main/java/org/eolang/compiler/syntax/Method.java

```diff
@@ -57,7 +57,8 @@
      * @param args Arguments
      * @param rtp Return type
      */
-    Method(final String mtd, final Collection<Argument> args, final String rtp) {
+    public Method(final String mtd, final Collection<Argument> args,
```

Comment:
Same as `tpn`

 * `mtd` Just makes it easier to think about not using mnemonics.
 * `rtp` Just makes it easier to think about not using mnemonics

I understand keywords may be a problem, but mnemonics are also a problem.